client/state/query-client.ts

```diff
@@ -1,10 +1,22 @@
 import { throttle } from 'lodash';
-import { QueryClient } from 'react-query';
+import { QueryClient, QueryCache } from 'react-query';
 import { persistQueryClient } from 'react-query/persistQueryClient-experimental';
+import { BASE_STALE_TIME } from 'calypso/data/marketplace/constants';
```

Comment:
`BASE_STALE_TIME` is no longer just a `marketplace` constant. Would be nice to move it elsewhere sooner or later.